typescript/infra/config/environments/mainnet3/gasPrices.json

```diff
@@ -159,10 +159,6 @@
     "amount": "0.001000252",
     "decimals": 9
   },
-  "fractal": {
```

Comment:
this one we left out of the batch yes? i see `No SAFE noted. No easy way to bridge gas tokens.`

Reply:
yup - we encountered some issues at deploy time with the provided RPC. Will debug with the team and roll into next batch